src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -86,12 +86,12 @@ private void testCreateInstructorAction() throws Exception {
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
I think there is some misunderstanding here.
My idea is to include the verification inside the retry method rather then retrying to get data.
For example, here we could write.
``` java
// comment here e.g. Instructor entity need time to be updated in GAE
RetryManager.run(new Retry {
  @Overribe
  public void try() {
    homePage.refresh();
    InstructorAttributes instructorInBackend = BackDoor.getInstructorByEmail(instructor.email, demoCourseId);
    assertEquals(String.format(Const.StatusMessages.INSTRUCTOR_DETAILS_LENGTH_INVALID,
                                     Const.LENGTH_FOR_NAME_EMAIL_INSTITUTION),
                       homePage.getMessageFromResultTable(1));
  }
});
```
Then, the interval could be 2-3 seconds, and the maxium attemp could be 4-5 times.